bp_top/test/tb/bp_tethered/Makefile.vcs

```diff
@@ -1,7 +1,17 @@
 VCS_TIMEOUT = 1000ms
 
-$(LINT_DIR)/testbench.sv $(LINT_DIR)/wrapper.sv $(LINT_DIR)/test_bp.sv:
+export SIM_SYNTH_VIVADO_P ?= 0
+export SIM_SYNTH_DC_P ?= 0
```

Comment:
two definitions for SIM_SYNTH_VIVADO? I think you can get rid of the ones in the tb/Makefile.params. Should be able to export in the target-specific variables for sim_vivado.v

Reply:
Removed the definition from this file. However, I had to add it in Makefile.params because the NO_BIND parameter uses the SIM_SYNTH_VIVADO_P parameter so was giving syntax errors when I removed it. Now, the only definition is in Makefile.params and the assignment is through the command. I tried adding it to the target specific variables for sim_vivado but it did not substitute the value correctly.